controllers/example_controller_python/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM uobflightlabstarling/starling-controller-base
+ARG VERSION=latest
+ARG REGISTRY
```

Comment:
Does Registry need a default argument too? or will it default to empty string?

Reply:
The docs aren't super clear on that. But it appears to behave as though the default is an empty string